src/plugins/foxPage/hooks/getGovernanceData.ts

```diff
@@ -28,7 +28,7 @@ const parseGovernanceData = (governanceData: BoardroomGovernanceData) => {
       title,
       choices,
       results: choices.map((_, i) => ({
-        absolute: results[i] ? results[i].total : 0,
+        absolute: bnOrZero(results[i]?.total).toString(),
         percent: results[i] ? bnOrZero(results[i].total).div(totalResults).toString() : '0',
```

Comment:
Note: Since we can't mathematically divide by zero, we have to use a ternary here